app/index.html

```diff
@@ -142,7 +157,93 @@
                 </paper-material>
               </template>
             </section>
+            
+            <section data-route="workflow">
+              <paper-material elevation="1">
+                <p> The application aims to simplify tracking activities done by Google Experts.</p>
+                <p>To do so, the backend on Google App Engine implements a series of jobs to automatically grab and update activities:</p>
+                <ul>
+                  <li>Create/Update activities from Google+</li>
+                  <li>Update Impact on Youtube videos with the number of views</li>
+                </ul>
+                
+                <h2>Activities from Google + Stream</h2>
+                <p> 
+                  A daily task is in charge of obtaining new <b>public</b> Google+ posts for each Google Expert, and create a new activity 
+                  if the post has the proper #hashtags as explained in the "How is it used" section.<br>
+                  <br>
+                  In case a post has only some of the required hashtags, the activity is still created and available for the Expert to fix using this app.
+                  <br>
+                  Once a week, activities generated from Google+ posts are updated with the new values of +1s, reshares and comments (Social Impact)
+                </p>
+                <h2>Youtube Videos statistics</h2>
+                <p>
+                  During the weekly update of the activities, if the activity has a YouTube link attached, the impact for the #content metadata is automatically updated with the views on the video.
+                </p>
+              </paper-material>
+            </section>
+            
+            <section data-route="how-to-use">
+              <!-- Prepare elements to access the API -->
+              <discovery-api-elements
+                name="gdetracking"
+                version="v1.0b2"
+                app-id="omega-keep-406"
+                no-client
+                ></discovery-api-elements>
+              <!--Load Product Groups from the API-->
+              <gdetracking-product_group-list auto response="{{pgGroups}}"></gdetracking-product_group-list>
```

Comment:
this would need a limit="100" parameter to fetch the full list, the list operations in the backend default to 10 items.